backend/src/test/java/com/senior/project/backend/AbstractDomainObjectTest.java

```diff
@@ -0,0 +1,157 @@
+package com.senior.project.backend;
```

Comment:
This is a test for domain objects with the generated methods, any test that extends this class will have the tests below run automatically.